redirector/sync.py

```diff
@@ -31,7 +33,7 @@ def lambda_handler(event, context):
     while True:
         for item in scan_response["Items"]:
             count += 1
-            if item["box_file_id"] not in shared_file_ids:
+            if (item["box_file_id"] not in shared_file_ids) | (item['filepath'] not in shared_file_paths):
```

Comment:
I would use `or` here instead of the bitwise `|`, since we're talking boolean logic in this case.

Reply:
fixed in bac6b2b